website/views/options/social/index.jade

```diff
@@ -39,7 +39,7 @@ script(type='text/ng-template', id='partials/options.social.party.html')
       p
         =env.t('startLFG')
         |&nbsp;
-        a(href="http://habitrpg.wikia.com/wiki/Board:The_Archery_-_LFG")=env.t('linkLFG')
+        a(href="https://habitrpg.com/#/options/groups/guilds/f2db2a7f-13c5-454d-b3ee-ea1f5089e601")=env.t('linkLFG')
```

Comment:
No, since we're keeping the Archery on the wiki, I don't think we should
change the name.

Let's say "To advertise your new party or find one to join, go to the Party
Wanted (Looking for Group) Guild."

On Tue, Feb 17, 2015 at 9:14 AM, Blade Barringer <notifications@github.com>
wrote:

> In website/views/options/social/index.jade
> <https://github.com/HabitRPG/habitrpg/pull/4702#discussion_r24832721>:
>
> > @@ -39,7 +39,7 @@ script(type='text/ng-template', id='partials/options.social.party.html')
> >        p
> >          =env.t('startLFG')
> >          |&nbsp;
> > -        a(href="http://habitrpg.wikia.com/wiki/Board:The_Archery_-_LFG")=env.t('linkLFG')
> > +        a(href="https://habitrpg.com/#/options/groups/guilds/f2db2a7f-13c5-454d-b3ee-ea1f5089e601")=env.t('linkLFG')
>
> Currently the title of the Guild is "Party Wanted (Looking for Group)"
>
> @lemoness <https://github.com/lemoness> Do you want to rename the guild
> to include the word "Archery"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/4702/files#r24832721>.
>